runtime/manual/references/contributing/building_from_source.md

```diff
@@ -76,6 +76,15 @@ brew install llvm
 # Add /opt/homebrew/opt/llvm/bin/ to $PATH
 ```
 
+**WSL**
+```shell
+wget https://apt.llvm.org/llvm.sh
+chmod +x llvm.sh
+./llvm.sh 16
+```
```

Comment:
More out of curiosity: Why does llvm need to be installed this way rather than via `apt/apt-get`? Is the default llvm version in ubuntu too old?

Reply:
Yes. I faced issues installing clang 16 which is not readily available in Ubuntu 22.04 LTS repos.

folks have raised issues at stack overflow here https://askubuntu.com/questions/1473403/how-to-install-clang-16-on-22-04-lts